### DISCUSSION TITLE
Q: 请问 ipv6位置不显示正常吗？Q2: 点赞看不到数量 || Q: Is it normal that the ipv6 location is not displayed? Q2: I can’t see the number of likes

### DISCUSSION THREAD

**Discussion by nanqic** [walinejs/waline#2351] [Q&A] [answered]
### Describe the bug | 问题描述

后台看不到 ipv6位置，只看到 v4 的，是 我个人配置的问题吗？

看到文档说文章反映基于 ip，点赞过的，点赞总数怎么显示 0 呢？控制台看到有请求数据

### Where your waline deploy? | 服务部署在哪里？

Netlify

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I can't see the ipv6 location in the background, but only v4. Is it a problem with my personal configuration?

I saw the document saying that the article reflection is based on IP and has been liked. Why does the total number of likes show 0? The console sees request data

### Where your waline deploy? | Where is the service deployed?

Netlify

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
1. 实现如此
2. 确认是个 Bug，之后处理一下

---
1. To achieve this
2. Confirm it is a bug and deal with it later

**Reply by nanqic** [reply]
有计划实现显示 ipv6位置吗？是不是 ipv6的 geoip 文件太大的原因

---
Are there any plans to implement display of ipv6 location? Is it because the geoip file of ipv6 is too large
